.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+related tracking issue:
+<!--
+In case this PR is related to an unstable feature or an otherwise tracked effort,
+please link to the relevant tracking issue here. If you don't know of a related
+tracking issue or there are none, feel free to ignore this.
+
+This automatically references this PR in the tracking issue which makes it far easier
+to see the ongoing process on a given issue and also provides a list of the relevant
+work when summarizing a feature for stabilization.
```

Comment:
We could also remind people that they can use `r?` here to select a reviewer if there is someone who has been mentoring them or seems appropriate for the work?